flo-workflow/src/main/java/com/spotify/flo/BuilderUtils.java

```diff
@@ -41,11 +41,6 @@
  */
 class BuilderUtils {
```

Comment:
We can add more test cases for this class to get better coverage.

Reply:
Separate PR?